XenAdmin/SettingsPanels/PerfmonAlertOptionsPage.cs

```diff
@@ -145,7 +157,8 @@ public bool HasChanged
                 return ((_OrigEmailNotificationCheckBox != EmailNotificationCheckBox.Checked) ||
                         (_OrigEmailAddressTextBox != EmailAddressTextBox.Text) ||
                         (_OrigSmtpServerAddrTextBox != SmtpServerAddrTextBox.Text) ||
-                        (_OrigSmtpServerPortTextBox != SmtpServerPortTextBox.Text));
+                        (_OrigSmtpServerPortTextBox != SmtpServerPortTextBox.Text) ||
+                        (_OrigMailLanguageCodeComboBox != MailLanguageComboBox.SelectedValue.ToString()));

```

Comment:
Yes you are right. I made mistake for the "_OrigMailLanguageCodeComboBox" assignment, I should use SelectedValue.ToString()